<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="teOutput"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="h3">
      <item>
       <widget class="QPushButton" name="bAddInput">
        <property name="text">
         <string>Input Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="h4">
      <item>
       <widget class="QPushButton" name="bAddOutput">
        <property name="text">
         <string>Output Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leOutput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="h5">
      <item>
       <widget class="QPushButton" name="bPipeConfig">
        <property name="text">
         <string>Pipeline Config</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePipeConfig">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="h6">
      <item>
       <widget class="QPushButton" name="bTSConfig">
        <property name="text">
         <string>Timestream Config</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leTSConfig">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="h1">
      <item>
       <spacer name="hs1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bRunPipe">
        <property name="text">
         <string>Run Pipeline</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="h2">
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pbpl">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
